packages/maps/src/components/result/GoogleMapMarker.js

```diff
@@ -164,15 +159,15 @@ class GoogleMapMarker extends React.Component {
 				return (
 					<MarkerWithLabel
 						key={marker._id}
-						labelAnchor={new window.google.maps.Point(0, 30)}
+						labelAnchor={new window.google.maps.Point(0, 0)}
```

Comment:
Is there any reason for the change?

Reply:
@bietkul 
https://www.loom.com/share/0a95d9227a43443397551d8df009dbdc